app/Resources/views/committee/show.html.twig

```diff
@@ -5,10 +5,93 @@
+                <div class="committee-link committee-follow-disabled">
+                    Suivre ce comité
+                </div>
+                <a href="{{ path('app_membership_register') }}" id="committee-register-link">Adhérer à En Marche!</a>
+            {% elseif can_follow(committee) %}
+                <a href="{{ committee_path('app_committee_unfollow', committee) }}" class="committee-link committee-follow">
+                    Suivre ce comité
+                </a>
+            {% elseif can_unfollow(committee) %}
+                <a href="{{ committee_path('app_committee_unfollow', committee) }}" class="committee-link committee-unfollow">
+                    Quitter ce comité
+                </a>
+            {% endif %}
+        </div>
+    </section>
+
+    <section class="l__wrapper--medium">
+        {% if is_host %}
+            <ul id="committee-host-nav">
+                <li>
+                    Discussion
+                </li>
+                <li>
+                   <a href="{{ committee_path('app_commitee_list_members', committee) }}">
+                       Membres
+                   </a>
+                </li>
+                <li>
+                    <a href="{{ committee_path('app_committee_edit', committee) }}">
+                        Éditer le comité
+                    </a>
+                </li>
+            </ul>
+
+            <div id="comittee-host-menu">
+                {#
+                    TODO implement event message form
+                #}
+                <a href="#">Créer un événement</a>
+            </div>
+        {% endif %}
+    </section>
+
+    <section class="l__wrapper--medium">
+        <div class="committee-details">
+            <h4>{{ 'committee.members_count'|transchoice(committee_members_count) }}</h4>
+            <h5>À propos</h5>
+            <p id="committee-description">{{ committee.description }}</p>
+        </div>
+        <div class="committee-details">
+            <h5>Administration</h5>
+            <ul class="committee-hosts">
+                {% for host in committee_hosts %}
+                    <li class="committee-host">
+                        {{ host }}
+                        {% if is_host and app.user.equals(host) %}
+                            (vous)
+                        {% else %}
+                            <a href="mailto:{{ host.emailaddress }}">Contacter</a>
+                        {% endif %}
+                    </li>
+                {% endfor %}
+            </ul>
+        </div>
+        <div class="committee-details">
+            <h5>Localisation</h5>
+            {#
+                TODO add static image
+            #}
+        </div>
+        <div class="committee-details">
+            <h5>Partager</h5>
+            <a href="{{ committee.facebookPageUrl }}">Facebook</a>
+            <a href="{{ committee.googlePlusPageUrl }}">Google +</a>
+            <a href="https://twitter.com/{{ committee.twitterNickname }}">Twitter</a>
```

Comment:
Are you sure that the nickname is checked before (Twitter's requirements about the format) ? In other case, may require an URL encoding.